docker-gateway-rest-server

```diff
@@ -29,8 +29,8 @@ from gateway_code.config import control_node_class
 
 
 def manual_udev(env, clazz):
-    if env in os.environ and hasattr(clazz, 'TTY'):
-        cmd = 'ln -nfs %s %s' % (os.environ[env], clazz.TTY)
+    if env in os.environ and hasattr(clazz, 'tty'):
```

Comment:
This was something I found not very nice in #13. Why not rename `clazz` in `cls` ? or `node_cls` ?